constants.js

```diff
@@ -124,6 +124,8 @@ const checksWhitelist = {
   'oppiabot': {
     [openEvent]: [claCheck],
     [reopenEvent]: [],
+    [openEventGithubActions]: [claCheckGithubAction],
+    [reopenEventGithubActions]: [claCheckGithubAction],
```

Comment:
Above this is also enabled for `reopenEvent`, should we add that too?

Reply:
No, it doesn't have to be added to the `reopenEvent`. I have removed from above.